### ISSUE TITLE
Pauses after like limit is hit

### ISSUE THREAD

**Issue by limerick11** [instabot-py/instabot.py#2359]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x ] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [x ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] Are you running the code on your phone 
- [x ] If so, have you tried running the code on your local machine?

```
Include more details about the environment
```
#### Operating System? (include version)
- [ ] macOS
- [ ] Windows
- [x ] Rasbian Linux Raspberry PI (include flavour)
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

```
include the output of 'uname -a'
```
Linux raspberrypi 4.19.58-v7+ #1245 SMP Fri Jul 12 17:25:51 BST 2019 armv7l GNU/Linux
#### Python Version Requirement
- [ x] I am using Python 3.6 or above

#### Exact Python Version?
```
Include here the response of 'python --version' or 'python3 --version'
```
Python 2.7.16
Python 3.7.3

#### Pip Version?
```
Include here the response of 'pip --version' or 'pip3 --version'
```
pip 18.1 from /usr/lib/python2.7/dist-packages/pip (python 2.7)
pip 18.1 from /usr/lib/python3/dist-packages/pip (python 3.7)

#### Instabot Version?
```
Include here the response of 'instabot-py --version'
```
Installed version 0.5.7

#### How you use the instabot
- [x ] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [ ] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue
I get the following after it hits the like limit and no more likes.
2019-08-07 23:59:00,467 -  - InstaBot - INFO - Pause for 10800 seconds
2019-08-08 23:59:00,256 -  - InstaBot - INFO - Pause for 10800 seconds
2019-08-09 23:59:00,440 -  - InstaBot - INFO - Pause for 10800 seconds
2019-08-10 23:59:00,441 -  - InstaBot - INFO - Pause for 10800 seconds

```
Include the contents of the error.log here (if applicable). < REPLACE THIS TEXT
```

#### Config

error_400_to_ban: 3
follow_per_day: 0
follow_per_run: 0
follow_time_enabled: false
like_per_day: 1000
like_per_run: 1000
log_mod: 1
max_like_for_one_tag: 89
media_max_like: 400
media_min_like: 0
proxies: {}
session_file: '{{login}}.session'
start_at_h: 0
start_at_m: 0
tag_blacklist:
- gym
tag_list:
- like
time_till_unlike: 259200
unfollow_break_max: 30
unfollow_break_min: 15
unfollow_everyone: false
unfollow_inactive: true
unfollow_not_following: false
unfollow_per_day: 0
unfollow_per_run: 0
unfollow_probably_fake: true
unfollow_recent_feed: false
unfollow_selebgram: false
unfollow_whitelist: []
unlike_per_day: 0
unlike_per_run: 0
unwanted_username_list: []
user_blacklist: {}
user_max_follow: 0
user_min_follow: 0
window_check_every: 300
```

**Comment by H00kahMan** [comment]
Check instabot.py, search 10800, change it to 0 maybe, I don't know for sure tbh.

**Comment by nickgryg** [comment]
This issue has been fixed in version `0.7.4`
